com.getstation.Station.yaml

```diff
@@ -0,0 +1,50 @@
+id: com.getstation.Station
+base: org.electronjs.Electron2.BaseApp
+base-version: '23.08'
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: station
+separate-locales: false
+rename-desktop-file: station-desktop-app.desktop
+rename-icon: station-desktop-app
+finish-args:
+  - --share=ipc
+  - --device=dri
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --env=ELECTRON_TRASH=gio
+modules:
+  - name: station-desktop
+    buildsystem: simple
+    build-commands:
+      - ar x Station-*.deb
+      - rm -f Station-*.deb
+      - tar xf data.tar.xz
+      - rm -f control.tar.gz data.tar.xz debian-binary
+      - cp -r usr/* opt/* /app
+      - chmod -R a-s,go+rX,go-w /app/Station
+      - rm -r /app/share/icons/hicolor/1024x1024
+      - desktop-file-edit --set-key=Exec --set-value='station %U' /app/share/applications/station-desktop-app.desktop
+      - install station.sh /app/bin/station
+      - install -Dm644 com.getstation.Station.metainfo.xml /app/share/metainfo/com.getstation.Station.metainfo.xml
+    sources:
+      - type: file
+        only-arches:
+          - x86_64
+        url: https://github.com/getstation/desktop-app/releases/download/v3.3.0/Station-amd64.deb
```

Comment:
so what happen in the aarch64 package?

Reply:
it's not ready yet

Reply:
that doesn't answer the question.

Reply:
I haven't had an opportunity to check it on aarch64. 
Arch constraint removed

Reply:
so what happen now when building the aarch64 package?

Reply:
no idea. I'm here to publish current release to flathub. 
current release doesn't contain dedicated aarch64 package.
is it a stopper?

Reply:
https://docs.flathub.org/docs/for-app-authors/maintenance#flathubjson

Reply:
or you could build from source.

Reply:
what are the blockers to a source build? Did you try?

Reply:
> what are the blockers to a source build? Did you try?

https://github.com/flathub/flathub/pull/6183#discussion_r1962244763

Reply:
no need to circle back to a non answer. 

The question is clear: what prevents from building from source?